client/packages/purchasing/src/purchase_order/api/hooks/usePurchaseOrderLine.ts

```diff
@@ -57,7 +57,7 @@ export function usePurchaseOrderLine(id?: string | null) {
   const { data, isLoading, error } = useGet(id ?? '');
 
   const { patch, updatePatch, resetDraft, isDirty } =
-    usePatchState<DraftPurchaseOrderLine>(data?.nodes[0] ?? {});
+    usePatchState<DraftPurchaseOrderLine>({});
```

Comment:
I think there was a previous misunderstanding of what the "patch" state is -- it should never need to have the whole draft value in it.